README.rst

```diff
@@ -69,6 +69,33 @@ VIA_BASE_URL
   don't have the Hypothesis Chrome extension installed
   (default: https://via.hypothes.is)
 
+Route Syntax/API
+----------------
+
+Share Annotations on Page/URL (``/go``)
++++++++++++++++++++++++++++++++++++++
+
+See the annotations at a URL, and optionally filter or add context to the
+annotations displayed at that URL.
+
+Querystring parameters:
+
+* ``url`` (required): URL of page/document to share
+* ``group``: Optional group ID, to share the URL in the context of a particular group
+* ``q``: Optional query string with which to pre-populate the client's search filtering
+
+
+Share an Annotation (``/{id}``)
++++++++++++++++++++++++++++++
+
+Go to an individual annotation, where ``id`` is the annotation's unique ID.
+
+Share an Annotation, with URL Information  (``/{id}/url``)
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
`/{id}` and `/{id}/url` are effectively the same route. Perhaps it might make sense to fold this into one heading, eg:

```
Share an annotation (``/{id}`` or ``/{id}/{url}``)
```